test/testdata/resolver/optional_nil.rb.flatten-tree.exp

```diff
@@ -68,15 +73,6 @@ begin
         ::Sorbet::Private::Static.sig(<self>) do ||
           <self>.params({:"w" => ::String}).returns(::String)
         end
-        <self>.extend(::T::Sig)
-        ::Sorbet::Private::Static.keep_def(<self>, :"foo")
-        ::Sorbet::Private::Static.keep_def(<self>, :"bar")
-        ::Sorbet::Private::Static.keep_def(<self>, :"qux")
-        ::Sorbet::Private::Static.keep_def(<self>, :"baz")
-        ::Sorbet::Private::Static.keep_def(<self>, :"foo<defaultArg>1")
-        ::Sorbet::Private::Static.keep_def(<self>, :"bar<defaultArg>1")
-        ::Sorbet::Private::Static.keep_def(<self>, :"qux<defaultArg>1")
-        ::Sorbet::Private::Static.keep_def(<self>, :"baz<defaultArg>1")
```

Comment:
This change is obviously bad and should not be merged. We need to keep these `keep_def` calls around for the `<defaultArg>` methods too. I was just curious to see whether we could take a quick-and-dirty approach to fixing this by running DefaultArgs after Flatten, which would have been a cute trick.